docs_app/content/tutorial/7-flattening-operations.md

```diff
@@ -0,0 +1,132 @@
+
+## Concat (concat, concatMap, concatAll, concatWith)
+
+Concat should be the go-to for most people's flattening needs in general RxJS use. This strategy will subscribe to one and only one inner observable at a time, queuing up any other inner observables that arrive during that inner [subscription](GL). When that inner subscription [completes](GL), the next one is dequeued and subscribed to.
+
+### When to use
+
+- Whenever order matters. If order is so important that you're willing to defer when each inner observable is subscribed to in order to make sure the results do not come back out of order, concat is the right choice.
+- Any HTTP request where all requests must be made, and the responses matter. If you're making a series of HTTP requests, and you need to make sure the responses are processed in order, you should use a concat strategy.
+
+### When NOT to use
+
+- Never use a concat strategy with extremely long-lived inner observables, or inner subscriptions that never complete. If it never completes, it cannot concatenate. Imagine another universe where you were trying to concat an array that was _infinitely long_ with any other array. You never get to where you're adding the contents of the second array because you cannot finish reading the first. Observables that never end cannot be concatenated.
+- When you have inner subscriptions you want to start right away. If this is the case, you'll probably want to use a [merge](#merge-merge-mergemap-mergeall-mergewith) or even a switch strategy.
+
+## Switching (switchMap, switchAll)
+
+Switching is a strategy that gets a decent amount of hype in RxJS tutorials. This comes from it's implicit cancellation behavior. This strategy will subscribe to one and only one inner observable at a time, but will subscribe to it as soon as it arrives. In order to ensure there is only one active inner subscription, it will unsubscribe from the previous inner subscription when the new one arrives. The switch strategy is the only strategy that will automatically unsubscribe from inner subscription to start the next one.
+
+It's called the "switch" strategy because it "switches" from whatever inner subscription it has to the next one it gets, as soon as it gets it.
+
+### When to use
+
+- Switch is particularly useful when dealing with situations where you want start the next subscription right away, and you do not care about the values of the previous one any more.
+- HTTP GETs involved with things like lookahead searches, or user-initiated HTTP GETS. It starts the request right away, and disregards the previous one, as you may not care about it.
+- Toggling between different observables. Let's say you had an animation loop and you wanted to turn it on and off, you could toggle between your animation loop observable and an empty observable using a switch strategy.
+
+### When NOT to use
+
+- Never use switchMap when the results that come back from your requests matter.
```

Comment:
This is a strong statement that could be misconstrued.  I have a list of products. The user clicks one to get the details. I use `switchMap` to execute the Http get to get the product the user just clicked on. The result that comes back from the server *very much* matters. Up until the user picks another product.

Maybe include `prior` in the sentence?

"Never use switchMap when the *prior* results that come back from your requests matter."